concepticondata/conceptlists/Zhou-2020-350.tsv

```diff
@@ -0,0 +1,351 @@
+Zhou-2020-350-282	282	to plant, to stick in	插、栽	1486	PLANT (SOMETHING)
+Zhou-2020-350-283	283	to pound	舂	354	POUND
+Zhou-2020-350-284	284	to press	壓	1913	PRESS
+Zhou-2020-350-285	285	to puke	嘔吐	1278	VOMIT
+Zhou-2020-350-286	286	to pull	拔	1455	PULL
+Zhou-2020-350-287	287	to rain	下（雨）	1253	RAIN (RAINING)
+Zhou-2020-350-288	288	to release	放	1095	LET GO OR SET FREE
+Zhou-2020-350-289	289	to rest	休息	168	REST
+Zhou-2020-350-290	290	to rise (water)	漲		
+Zhou-2020-350-291	291	to roast	烤	1962	ROAST (SOMETHING)
+Zhou-2020-350-292	292	to roll	滾	827	ROLL
+Zhou-2020-350-293	293	to scrape	刮	1767	SCRAPE
+Zhou-2020-350-294	294	to see	看見	1409	SEE
+Zhou-2020-350-295	295	to send away	送（走）		
+Zhou-2020-350-296	296	to sew	縫	1457	SEW
+Zhou-2020-350-297	297	to sit	坐	1649	SIT DOWN
+Zhou-2020-350-298	298	to snow	下（雪）	2234	FALL (OF SNOW AND RAIN)
+Zhou-2020-350-299	299	to soak	浸泡	2069	SOAK (VERB)
+Zhou-2020-350-300	300	to split	劈	3151	CUT (WITH AXE)
+Zhou-2020-350-301	301	to spread out	鋪開	38	SPREAD OUT
+Zhou-2020-350-302	302	to squat	蹲	296	CROUCH
+Zhou-2020-350-303	303	to stand	站	1442	STAND
+Zhou-2020-350-304	304	to steal	偷	713	STEAL
+Zhou-2020-350-305	305	to steam	蒸	2067	STEAM (VERB)
+Zhou-2020-350-306	306	to step on	踩	575	TREAD
+Zhou-2020-350-307	307	to stink	臭	42	STINKING
+Zhou-2020-350-308	308	to store up	收藏	3655	STORE UP
+Zhou-2020-350-309	309	to swallow	吞	897	SWALLOW
+Zhou-2020-350-310	310	to sweep	掃	230	SWEEP
+Zhou-2020-350-311	311	to swing	甩	2871	SWING (MOVEMENT)
+Zhou-2020-350-312	312	to think	想	2271	THINK
+Zhou-2020-350-313	313	to throw	扔	1456	THROW
+Zhou-2020-350-314	314	to tie together	捆		
+Zhou-2020-350-315	315	to tie up	綁	1917	TIE
+Zhou-2020-350-316	316	to touch	摸	1892	TOUCH
+Zhou-2020-350-317	317	to understand, to know	知道	1536	UNDERSTAND
+Zhou-2020-350-318	318	to wait	等待	1590	WAIT (FOR)
+Zhou-2020-350-319	319	to walk	走	1443	WALK
+Zhou-2020-350-320	320	to wash	洗	1453	WASH
+Zhou-2020-350-321	321	to wash	淘洗	1453	WASH
+Zhou-2020-350-322	322	to water (a plant)	澆	1487	POUR
+Zhou-2020-350-323	323	to wear	穿	976	WEAR
+Zhou-2020-350-324	324	to weave	織	133	WEAVE
+Zhou-2020-350-325	325	to wipe	擦	1454	WIPE
+Zhou-2020-350-326	326	to wrap	包	1113	WRAP
+Zhou-2020-350-327	327	to write	寫	1672	WRITE
+Zhou-2020-350-328	328	tongue	舌頭	1205	TONGUE
+Zhou-2020-350-329	329	tooth	牙	1380	TOOTH
+Zhou-2020-350-330	330	Tujia	土家		
+Zhou-2020-350-331	331	two	二	1498	TWO
+Zhou-2020-350-332	332	two (days) later	後天	1179	DAY AFTER TOMORROW
+Zhou-2020-350-333	333	urine	尿	1142	URINE
+Zhou-2020-350-334	334	vegetable	蔬菜	924	VEGETABLES
+Zhou-2020-350-335	335	village	村莊	930	VILLAGE
+Zhou-2020-350-336	336	water	水	948	WATER
+Zhou-2020-350-337	337	we	咱們	1131	WE (INCLUSIVE)
+Zhou-2020-350-338	338	what	什麼	1236	WHAT
+Zhou-2020-350-339	339	white	白	1335	WHITE
+Zhou-2020-350-340	340	who	誰	1235	WHO
+Zhou-2020-350-341	341	wild cat	野貓	2241	WILD CAT
+Zhou-2020-350-342	342	wind	風	960	WIND
+Zhou-2020-350-343	343	wine	酒	1524	WINE
+Zhou-2020-350-344	344	wok	鍋	2056	WOK
+Zhou-2020-350-345	345	written word	字		
```

Comment:
1599 WORD ?

Reply:
The Chinese character is typically translated as "sign", referring to one Chinese character, so I'd rather leave it as is.